docs/data/external-oasis-repos.md

```diff
@@ -142,37 +141,21 @@ Host a Repository on OASIS
 
     Replace the %RED%red%ENDCOLOR% items with the appropriate values.
 
-3.  (**OSG internal step**)
-    OSG Operations ensures that the repository administrator is valid for the VO. This can be done by (a) OSG
-    already having a relationship with the person or (b) the GOC representative contacting the VO manager to find out.
-    OSG Operations should review the URL to verify it is appropriate for the VO.  This typically takes one business day.
-
-4.  (**OSG internal step**)
-    The OSG representative adds the repository URL in OIM under the VO's OASIS repository URLs. This should cause
-    the repository's configuration to be added to the GOC Stratum-0 within 15 minutes.
-
-5.  (**OSG internal step**) If the respository ends in a new
-    domain name that has not been distributed before, the OSG representative next places a copy of the `domain.name.pub`
-    public key from `domain.name` into `/srv/etc/keys` on both `oasis-replica` and `oasis-replica-itb`. If the OSG
-    representative does not have that key, he or she will ask the repository service representative how to obtain it.
-    In order to support CVMFS client versions 2.2.X, a symbolic link of `domain.name.conf` has to be made in
-    `/cvmfs/config-osg.opensciencegrid.org/etc/cvmfs/domain.d` pointing to `default.conf`. This symbolic link has to be
-    created on the `oasis-itb` machine's copy of the `config-osg.opensciencegrid.org` repository and then copied to
-    production with the `copy_config_osg` command on the oasis machine.
-
-6.  If the repository name matches `*.opensciencegrid.org` or `*.osgstorage.org`, the OSG representative responds in
-    the ticket to ask that step \#6 be done; all other repositories (such as `*.egi.eu`) skip this step.
+1.  If the repository name matches `*.opensciencegrid.org` or `*.osgstorage.org`, wait for the go-ahead from the OSG
+    representative before continuing with the remaining instructions; for all other repositories (such as `*.egi.eu`),
+    you are done.
 
-    The repository administrator should **execute the following commands**:
+1. Execute the following commands:
```

Comment:
I think it would be good to repeat here about the wait, for example: "Once you are told in the ticket to proceed to the next step, execute the following commands:"